### PR TITLE
Issue 2869

### LINKED ISSUE #2869: Various updates to our Project Team pages
I am preparing various updates to our Project Team pages.

**Objective 1**:

- [x] To adjust the instruction _Please direct correspondence in the first instance to_  so that initial, general correspondence is directed to admin [@] programminghistorian.org (where it will be re-directed to MEs only where necessary).

To achieve this, I will update the following pages:

/en/project-team.md
/es/equipo-de-proyecto.md
/fr/equipe-projet.md
/pt/equipe.md

removing the liquid syntax reading `{% include managing-editor-loop.html %}`, and replacing it with `<a href="mailto: ">TEXT</a>` link.

**Objective 2**:

- [x] To update Team Roles and descriptions for ProgHist Ltd, so that the labels specify the actual role (e.g., "Trustee") and the descriptions better reflect those roles.

To achieve this, I will update`teamroles.yml` making adjustments to Team Roles and descriptions, lines 43-53.

I will also update `ph_authors.yml` to reflect this, as well as other staffing changes.

---

I'll prepare these changes first (just to make sure everything works!) and then seek input for the team to check/adjust my translations.

### PR DESCRIPTION
**Pull request by anisa-hawes**
I'm preparing various updates to the Project Team pages

Closes #2869

- Update initial contact across Project Team pages
- Adjust labels and definitions  for `- type: proghist`
- Add new role `- type: - publishing-manager` + labels and definitions
- Add new role `- type: proghist-chair` + labels and definitions
- Add loop for `publishing-manager`
- Add loop for `proghist-chair`
- Adjust Adam's `team_roles:` to define `- proghist-chair`, line 49.
- Adjust my `team_roles:` to define `- publishing-manager`, line 2196
- Adjust Zoe's `team_roles:` to define `- proghist` at line 1405
- Add `team_end: 2023` line 2654 to remove Gisele from the Project Team
- Update Adam's profile portrait
--
- [x] First, I'm checking how/whether any of this works ... (with huge thanks to @jenniferisasi)
- [x] Next, I'll be seeking input from the team to check/adjust my translations
--

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Add the appropriate "Label"
- [x] If this PR closes an Issue, add the phrase `Closes #ISSUENUMBER` to your summary above
- [x] Ensure the status checks pass: if you have difficulty fixing build errors, please contact our Publishing Assistant @anisa-hawes 
- [x] Check the Netlify Preview: navigate to netlify/ph-preview/deploy-preview and click 'details' (at right)
- [x] Assign at least one individual or team to "Reviewers"
  - [x] if the text needs to be translated, please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines), then assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing editor in your PR.

### CODE REVIEW

**Comment by anisa-hawes** [comment]
Thank you for these edits, @rivaquiroga!

There are a few aspects I still need to check... and a broken link to fix too, I think...!

**Comment by anisa-hawes** [comment]
Trying to unpick what I have done wrong here:

- I've made my profile image disappear from the Project Team page
- I've made Zoe's profile image appear twice in the ProgHist Ltd team
- I’m confused about how Zoe’s `sortname:` is working. It is set as LeBlanc [L] but it appears at the end of the alphabetical list, following [S] for Sichani
- Also, I notice on the live website that Zoe's profile doesn’t have the label ProgHist Ltd, yet she still appears in that team

**Comment by anisa-hawes** [comment]
- Build is failing due to broken links. These are being fixed in #2872.
- Other problems still to resolve !

**Comment by jenniferisasi** [comment]
So happy right now, @anisa-hawes! My read on the issue worked :) All had to do with the [project-team-loop.html](https://github.com/programminghistorian/jekyll/blob/gh-pages/_includes/project-team-loop.html) file and how it is calling on people to appear on each group on the Project Team page, only from certain tags. 


- I've made my profile image disappear from the Project Team page

> There was no loop on the .html looking for the 'publishing-manager' tag; so the moment you removed 'prog-hist' from your team role, it wouldn't know you were there

- I've made Zoe's profile image appear twice in the ProgHist Ltd team
> Because there was a loop on the .html for 'tech-lead' role, and now that you added 'prog-hist' to Zoe's profile, it was creating two profiles for her, so to speak, because it was two different loops - I deleted the 'tech-lead' loop to avoid this in the future; if someone leads the role and is not part of 'prog-hist' we would need to add it again

- I’m confused about how Zoe’s sortname: is working. It is set as LeBlanc [L] but it appears at the end of the alphabetical list, following [S] for Sichani
> This I'm not sure about, but I believe you go under everyone regardless of ABC's of sort name, because it looks for 'prog-hist' first, and then adds 'publishing-manager' loop, and so forth.

- Also, I notice on the live website that Zoe's profile doesn’t have the label ProgHist Ltd, yet she still appears in that team
> She appeared on the ProgHist Ltd because of the 'tech-lead' loop within ProgHist Ltd in the [project-team-loop.html](https://github.com/programminghistorian/jekyll/blob/gh-pages/_includes/project-team-loop.html) file


So, basically, the file that is looking for people and their team roles to put them in each team is the [project-team-loop.html](https://github.com/programminghistorian/jekyll/blob/gh-pages/_includes/project-team-loop.html) file, and whenever a new role is added or changed, a change needs to take place in it too. Yay! And congrats on your new role :)

**Comment by anisa-hawes** [comment]
Amazing, @jenniferisasi! 🌟

**Comment by anisa-hawes** [comment]
Dear @jenniferisasi. Many thanks for your help!

I have made a few additional tweaks, including:

- Adding a new loop for `proghist-chair`
- Adding a new role `- type: proghist-chair` + labels and definitions
- Re-ordering the loops for ProgHist Ltd roles, so that `proghist-chair` precedes `proghist`, and that precedes `publishing-manager`

-- 
One further small but perhaps useful (?) adjustment would be to update the Contents section, so that _ProgHist Ltd_ instead reads: _Programming Historian's publisher: ProgHist Ltd_. 

Q: From where is the `{% include toc.html %}` on the Project Team pages generated? Would this adjustment be made by changing the heading `### ProgHist Ltd` at line 36 of `_includes/project-team-loop.html`? 

I think the complexity of doing this may be that at the moment, the contents list is not translated. So maybe for now, I will simply take a note about this for when we redesign the website.

**Comment by anisa-hawes** [comment]
Dear @marie-flesch and @DanielAlvesLABDH,

I hope you're well?

I'd like to ask for your (or a member of your teams') support to check my translations for three`team_roles:` types alongside their labels and definitions. (Two labels are new `proghist-chair` + `publishing-manager`, and one `proghist` has been adjusted) 

```
- type: proghist-chair
  label:
    en: Chair of Trustees
    es: Presidente/a de Consejeros
    fr: Président/e du Fiduciaire
    pt: Presidente/a do Fiduciário/a
  definition:
    en: This member holds the role of Chair of Trustees on the Board of the not-for-profit company ProgHist Ltd, which supports the publications.
    es: Este miembro funge el cargo de Presidente/a Consejo de Administración en la compañía sin fines de lucro ProgHist Ltd, que gestiona nuestras publicaciones.
    fr: Ce(tte) membre occupe un rôle Président/e du Fiduciaire au sein du conseil d'administration de la compagnie sans but lucratif ProgHist Ltd, qui soutient les publications.
    pt: Este membro tem um papel Presidente/a do Fiduciário/a no conselho de fiduciários na gestão da organização sem fins lucrativos ProgHist Ltd., que suporta a publicação.
```
```
- type: proghist
  label:
    en: Trustee
    es: Consejero/a
    fr: Fiduciaire
    pt: Fiduciário/a
  definition:
    en: This member holds a role on the Board of the not-for-profit company ProgHist Ltd, which supports the publications.
    es: Este miembro funge un cargo en el Consejo de Administración en la compañía sin fines de lucro ProgHist Ltd, que gestiona las publicaciones.
    fr: Ce(tte) membre occupe un rôle au sein du conseil d'administration de la compagnie sans but lucratif ProgHist Ltd, qui soutient les publications.
    pt: Este membro tem um papel no conselho de fiduciários na gestão da organização sem fins lucrativos ProgHist Ltd., que suporta a publicação.
```

```
- type: publishing-manager 
  label:
    en: Publishing Manager
    es: Directora de publicaciones
    fr: Responsable de la publication
    pt: Gestão da publicação
  definition:
    en: Responsible for the development and production of our publications
    es: Responsable del desarrollo y la producción de nuestras publicaciones
    fr: Responsable de l'élaboration et de la réalisation de nos publications
    pt: Responsável pelo desenvolvimento e produção das nossas publicações
```

I'm happy for you to make any corrections/adjustments directly to the `teamroles.yml` file [in this branch](https://github.com/programminghistorian/jekyll/pull/2870/files), or to add a comment here so I can make the changes you recommend.

Thank you!

& Thank you @rivaquiroga for checking and correcting the ES already ☺️

**Comment by anisa-hawes** [comment]
Merci, @marie-flesch!

**Comment by anisa-hawes** [comment]
Hello @DanielAlvesLABDH and @programminghistorian/portuguese-team,

I'd like to ask if you might have time to review the Portuguese text I drafted to translate three`team_roles:` types alongside their labels and definitions. Two of the labels are completely new (`proghist-chair` + `publishing-manager`), and one (`proghist`) has been edited.

```
- type: proghist-chair
  label:
    en: Chair of Trustees
    es: Presidente/a de Consejeros
    fr: Président/e du Fiduciaire
    pt: Presidente/a do Fiduciário/a
  definition:
    en: This member holds the role of Chair of Trustees on the Board of the not-for-profit company ProgHist Ltd, which supports the publications.
    es: Este miembro funge el cargo de Presidente/a Consejo de Administración en la compañía sin fines de lucro ProgHist Ltd, que gestiona nuestras publicaciones.
    fr: Ce(tte) membre occupe un rôle Président/e du Fiduciaire au sein du conseil d'administration de la compagnie sans but lucratif ProgHist Ltd, qui soutient les publications.
    pt: Este membro tem um papel Presidente/a do Fiduciário/a no conselho de fiduciários na gestão da organização sem fins lucrativos ProgHist Ltd., que suporta a publicação.
```
```
- type: proghist
  label:
    en: Trustee
    es: Consejero/a
    fr: Fiduciaire
    pt: Fiduciário/a
  definition:
    en: This member holds a role on the Board of the not-for-profit company ProgHist Ltd, which supports the publications.
    es: Este miembro funge un cargo en el Consejo de Administración en la compañía sin fines de lucro ProgHist Ltd, que gestiona las publicaciones.
    fr: Ce(tte) membre occupe un rôle au sein du conseil d'administration de la compagnie sans but lucratif ProgHist Ltd, qui soutient les publications.
    pt: Este membro tem um papel no conselho de fiduciários na gestão da organização sem fins lucrativos ProgHist Ltd., que suporta a publicação.
```

```
- type: publishing-manager 
  label:
    en: Publishing Manager
    es: Directora de publicaciones
    fr: Responsable de la publication
    pt: Gestão da publicação
  definition:
    en: Responsible for the development and production of our publications
    es: Responsable del desarrollo y la producción de nuestras publicaciones
    fr: Responsable de l'élaboration et de la réalisation de nos publications
    pt: Responsável pelo desenvolvimento e produção das nossas publicações
```

I'm happy for you to make any corrections/adjustments directly to the `teamroles.yml` file [in this branch](https://github.com/programminghistorian/jekyll/pull/2870/files), or to add a comment here so I can make the changes you recommend.

Thank you.
Anisa

**Comment by DanielAlvesLABDH** [comment]
@anisa-hawes I have made the revisions, including one or two small changes in other texts

**Comment by anisa-hawes** [comment]
Muito obrigado, @DanielAlvesLABDH.

([I am fixing the broken link](https://github.com/programminghistorian/jekyll/pull/2888), and will then be ready to request a final review from Adam and merge these changes in).

**Comment by anisa-hawes** [comment]
Hello @acrymble,

Do you have time to take a final glance / review this? (I'd like to be sure that you're happy with the labels).

Thank you,
Anisa

**Review by acrymble — APPROVED** [review]
